src/jdk.management.agent/share/conf/management.properties

```diff
@@ -302,5 +302,27 @@
 #   If the pattern ends with "*", it matches any class with the pattern as a prefix.
 #   If the pattern is equal to the class name, it matches.
 #   Otherwise, the status is UNDECIDED.
-com.sun.management.jmxremote.serial.filter.pattern=java.lang.*;java.math.BigInteger;java.math.BigDecimal;java.util.*;javax.management.openmbean.*;javax.management.Attribute;javax.management.ObjectName;java.rmi.MarshalledObject;javax.security.auth.Subject;!*
+#
+#   Ending with !* ensures we reject classes which are otherwise unmatched.
+com.sun.management.jmxremote.serial.filter.pattern=\
+java.lang.*;\
+java.lang.reflect.Proxy;\
+java.math.BigInteger;\
+java.math.BigDecimal;\
+java.util.*;\
+javax.management.*;\
+javax.management.modelmbean.*;\
+javax.management.monitor.*;\
+javax.management.openmbean.*;\
+javax.management.relation.*;\
+javax.management.remote.*;\
+javax.management.remote.rmi.*;\
+javax.management.timer.*;\
+javax.management.Attribute;\
```

Comment:
This is unnecessary with the OpenJDK changes, javax.management.* was added above.

Reply:
Removed the OpenJ9 patch.